resolver/src/p2p/fetcher.rs

```diff
@@ -104,7 +104,7 @@ impl<E: Clock + GClock + Rng + Metrics, P: Array, Key: Array, NetS: Sender<Publi
         let shuffle = !is_new;
         let Some((peer, id)) = self.requester.request(shuffle) else {
             // If there are no peers, add the key to the pending queue
-            warn!(?key, "requester failed");
+            debug!(?key, "requester failed");
```

Comment:
As a compromise, we could add a metric here instead (probably more informative than a log)?